vizsite-frontend/src/features/wesbsiteBuilder/components/helpers.jsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable no-param-reassign */
+import { v4 as uuidv4 } from 'uuid';
+
+export const getNodeByType = (type) => {
+  if (type === 'div') {
+    return { value: uuidv4(), type: 'div', label: 'Div' };
+  }
+  if (type === 'text') {
+    return { value: uuidv4(), type: 'text', label: 'Text' };
+  }
+  if (type === 'img') {
+    return { value: uuidv4(), type: 'img', label: 'Image' };
+  }
+  if (type === 'input') {
+    return { value: uuidv4(), type: 'input', label: 'Input' };
+  }
+  if (type === 'button') {
+    return { value: uuidv4(), type: 'button', label: 'Button' };
+  }
+  if (type === 'ul') {
+    return { value: uuidv4(), type: 'ul', label: 'List' };
+  }
+  if (type === 'li') {
+    return { value: uuidv4(), type: 'li', label: 'List Item' };
+  }
+  if (type === 'table') {
+    return { value: uuidv4(), type: 'table', label: 'Table' };
+  }
+  return null;

```

Comment:
do something like below instead of using mutliple if loops 
let obj = {
table:{value:uuidv4(),type:"table",label:"Table"}
}

access it's value using obj.table